fili-core/src/test/groovy/com/yahoo/bard/webservice/util/DimensionStoreKeyUtilsSpec.groovy

```diff
@@ -0,0 +1,41 @@
+// Copyright 2016 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.util
+
+import com.yahoo.bard.webservice.config.BardFeatureFlag
+import spock.lang.Specification
+
+class DimensionStoreKeyUtilsSpec extends Specification {
+
+    def "Test getRowKey() with CASE_SENSITIVE_KEYS_ENABLED off"() {
+        expect:
+        DimensionStoreKeyUtils.getRowKey("FOO","1") == "foo_1_row_key"
+    }
+
+    def "Test getColumnKey() with CASE_SENSITIVE_KEYS_ENABLED off"() {
+        expect:
+        DimensionStoreKeyUtils.getColumnKey("FOO") == "foo_column_key"
+    }
+
+    def "Test getRowKey() with CASE_SENSITIVE_KEYS_ENABLED on"() {
```

Comment:
Ideally, test names reflect something about the behavior they are testing / asserting. With that in mind, could we update the test names in this file to be a bit more informative about what condition / behavior they are asserting exists?

For this particular case, it might be something like `getRowKey retains the case of the rowName when CASE_SENSITIVE_KEYS_ENABLED is off`